### ISSUE TITLE
change default value of `focusAfterRun` to be `none` for screen reader users

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#191804]
Changing this behavior could be jarring, so instead, we will set it to `none` by default and include a hint about this in the terminal accessibility help dialog.

**Comment by meganrogge** [comment]
To verify, enable screen reader mode, run the command `Run Selected Text in Active Terminal`. The terminal shouldn't be focused and the accessible buffer shouldn't be opened. Change the setting and repeat and verify that it works per the setting descriptions.

**Comment by Challanger524** [comment]
So that is u - @meganrogge, who spoiled the terminal focus in _user-input_ scenarios.

Would be awesome if **auto-focus** worked only on **pending user input** to the terminal, so everybody would be happy.
But I have no clue if such a functionality is even possible.

**Comment by meganrogge** [comment]
not sure what you mean, but you can set it to `terminal` to have the terminal focused automatically

![Screenshot 2024-02-05 at 12 33 47 PM](https://github.com/microsoft/vscode/assets/29464607/c4626b7d-cbd6-4118-98b5-29b2bf13129b)

**Comment by Challanger524** [comment]
Yes thank you, I figured that before. I mean next cases at the same time:
1. program only prints to the console - no focus to the `terminal`
2. program waits for user input - terminal receives _auto-focus_
